lib/pages/user_profile.dart

```diff
@@ -0,0 +1,185 @@
+                  ),
+                  ),
+                  Text(
+                    '100k',
+                    style: TextStyle(
+                  fontSize: 18.0, color: Colors.white),
+                  )
+                ],
+              ),
+              Row(
+                children: <Widget>[
+                  Container(
+                    padding: const EdgeInsets.all(0.0),
+                    width: 32.0,
+                    child:
+                  Icon(
+                    Icons.favorite,
+                  color: DefaultColors.iconColor
+                  ),
+                  ),
+                  Text(
+                    '100k',
+                    style: TextStyle(
+                  fontSize: 18.0, color: Colors.white),
+                  )
+                ],
+              ),
+              Row(
+                children: <Widget>[
+                  Container(
+                    padding: const EdgeInsets.all(0.0),
+                    width: 32.0,
+                    child:
+                  Icon(
+                    Icons.comment,
+                  color: DefaultColors.iconColor
+                  ),
+                  ),
+                  Text(
+                    '100',
+                   style: TextStyle(
+                  fontSize: 18.0, color: Colors.white),
+                  )
+                ],
+              ),
+            ],
+          ),
+        ),
+        Padding(
+          padding: const EdgeInsets.all(0.0),
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: <Widget>[
+              Row(
+                children: <Widget>[
+                  Container(
+                    padding: const EdgeInsets.all(24.0),
+                    child:
+                  MaterialButton(
+                    child: Text('Add new recipe'),
+                    color: DefaultColors.iconColor,
+                    textColor: DefaultColors.textColor,
+                    shape: RoundedRectangleBorder(
+                      borderRadius: new BorderRadius.circular(18.0),
+                      side: BorderSide(color:DefaultColors.iconColor),
+                    ),
+                    onPressed: (){},
+                  ),
+                  ),
+                ],
+              ),
+              Row(
+                children: <Widget>[
+                  Container(
+                    padding: const EdgeInsets.all(24.0),
+                    child:
+                  MaterialButton(
+                    child: Text('Manage my recipes'),
+                    color: DefaultColors.iconColor,
+                    textColor: DefaultColors.textColor,
+                    shape: RoundedRectangleBorder(
+                      borderRadius: new BorderRadius.circular(18.0),
+                      side: BorderSide(color:DefaultColors.iconColor),
+                    ),
+                    onPressed: (){},
+                  ),
```

Comment:
These rows are the same. Extract them to a method.

Reply:
extracted to new method: buildUserButtons